frontend/pages/knowledge-hub.js

```diff
@@ -334,36 +351,57 @@ export default function KnowledgeHub() {
                           }}
                         />
                       ) : filterType === 'date' ? (
-                        <div className="bg-white rounded-md shadow-lg border w-48 max-h-60 overflow-y-auto">
-                          {filterOptions[filterType].map((option) => (
-                            <label
-                              key={option}
-                              className="flex items-center px-4 py-2 hover:bg-gray-50 cursor-pointer text-black"
-                            >
-                              <input
-                                type="radio"
-                                name="date-filter"
-                                checked={
-                                  activeFilters[filterType].length > 0
-                                    ? activeFilters[filterType][0] === option
-                                    : option === 'All Time'
-                                }
-                                onChange={() => {
-                                  const newFilters = {
-                                    ...activeFilters,
-                                    [filterType]:
-                                      option === 'All Time' ? [] : [option],
-                                  };
-                                  setActiveFilters(newFilters);
-                                  updateFilters(newFilters);
-                                  setOpenFilter(null);
-                                }}
-                                className="mr-2"
-                              />
-                              {option}
-                            </label>
-                          ))}
-                        </div>
+                        <CheckboxFilter
```

Comment:
I can't actually immediately spot where this is happening in code to comment on that bit but it would be good to have the dropdown button to select the year labelled as "Year" rather than date. The sort can stay Date is the behaviour is more granular.